app/services/rate_limit_checker.rb

```diff
@@ -85,10 +86,17 @@ def check_comment_creation_limit
   end
 
   def check_published_article_creation_limit
+    # TODO: Vaidehi Joshi - We should make this time frame configurable.
     user.articles.published.where("created_at > ?", 30.seconds.ago).size >
       SiteConfig.rate_limit_published_article_creation
   end
 
+  def check_published_article_antispam_creation_limit
+    # TODO: Vaidehi Joshi - We should make this time frame configurable.
+    user.articles.published.where("created_at > ?", 5.minutes.ago).size >
```

Comment:
We should make this `5.minutes.ago` a configurable timeframe; but for now, just hardcoded it and left a note for myself to do that later :)